internal/driver/readhandler.go

```diff
@@ -103,12 +103,19 @@ func makeReadRequest(deviceClient *opcua.Client, req sdkModel.CommandRequest) (*
 
 	// make new result
 	reading := resp.Results[0].Value.Value()
-	return newResult(req, reading)
+
+	result, err := newResult(req, reading)
+
+	// get source timestamp
+	sourceTimeStamp := extractSourceTimestamp(resp.Results[0])
+	result.Tags["source timestamp"] = sourceTimeStamp.String()
+
+	return result, err
 }
 
-func makeMethodCall(deviceClient *opcua.Client, req sdkModel.CommandRequest) (*sdkModel.CommandValue, error) {
+func (d *Driver) makeMethodCall(deviceClient *opcua.Client, req sdkModel.CommandRequest) (*sdkModel.CommandValue, error) {
 	var inputs []*ua.Variant
-
+	d.Logger.Infof("enter makeMethodCall: ")
```

Comment:
Here is still one of the info loggings left!?

Reply:
done